src/test/scala/karazin/scala/users/group/week3/homework/ProgramSuite.scala

```diff
@@ -13,8 +19,10 @@ import scala.concurrent.Future
 
 class ProgramSuite extends munit.FunSuite:
   
-  test("successful async test example") {
-    Future {
-      assertEquals(42, 42)
-    }
+  test("getPostView async test") {
+      val postView = getPostView(Post(UUID.randomUUID(),UUID.randomUUID()))
+      postView onComplete {
```

Comment:
`onComplete` is a callback function, it means that the test could pass but the conditions in the callback wasn't executed at all due to async behaviour. Please use https://scalameta.org/munit/docs/tests.html#declare-async-test approach.